Applications/spire/include/spire/time_and_sales/time_and_sales_properties_dialog.hpp

```diff
@@ -64,7 +63,6 @@ namespace spire {
       QString m_band_list_stylesheet;
```

Comment:
Remove m_band_list_stylesheet. In general don't store the stylesheet as a member, especially if it never changes. Instead compute it when you actually need it.

Reply:
I combine it with the selected style to add the selected item border.

Reply:
A class field should be unique to every instance of that class, meaning that something about that field depends on the object to which it's a member of. Reviewing the code I don't see anything that m_band_list_stylesheet depends on, it's a constant that is the same from every single instance. As a general principle if this is the case then the options are as follows (in order of preference):

P1. Compute the value if and only when it's needed.

If P1 is too expensive because the value is used frequently, then:

P2. Store the value in a static const variable whose scope is as narrow as possible.